remotespark/kernels/sparkkernel/sparkkernel.py

```diff
@@ -13,6 +13,7 @@ def __init__(self, **kwargs):
         language_info = {
             'name': 'scala',
             'mimetype': 'text/x-scala',
+            'codemirror_mode': 'text/x-scala',
```

Comment:
This is not a regression, this line was never present in the file.

I am unsure why it didn't fail before, though this seems to be the correct way to set the kernel's language to Scala.